src/environments/environment.ts

```diff
@@ -3,7 +3,16 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  firebaseConfig: {
+    apiKey: 'AIzaSyBOv1qnMLeOrEumB2KXe4MCwtlf9nM_l7c',
```

Comment:
Never publish credentials in a repository specially if it is public

Reply:
@AntonioVarela look at the answer here: https://stackoverflow.com/questions/54061833/production-credentials-with-angular-6 ... it looks useful for us